Polygon_mesh_processing/doc/Polygon_mesh_processing/NamedParameters.txt

```diff
@@ -319,6 +319,23 @@ Monte-Carlo methods.
 \b Default value is `0`
 \cgalNPEnd
 
+\cgalNPBegin{do_project} \anchor PMP_do_project
+Parameter used in `random_perturbation()` to set whether vertices should be re-projected
+to the input surface after their geometric perturbation.
+\n
+\b Type : `bool` \n
+\b Default value is `true`
+\cgalNPEnd
+
+\cgalNPBegin{random_seed} \anchor PMP_random_seed
+Parameter used in `random_perturbation()` to choose a seed for the random number generator
+`CGAL::Random()`. If this parameter is not used, perturbation is not deterministic
+(i.e. not reproducible from one run to the other).
+\n
+\b Type : `unsigned int` \n
+\b Default is that this variable is unset
```

Comment:
I think unset and non set is not the same thing. Anyway, I guess it is initialized to some seed.